definitions/variable/sdg-indicators/sdg.yaml

```diff
@@ -31,6 +31,14 @@
       gas)
     unit: million
     navigate: Population|Clean cooking Access
+- Population|Sanitation Access:
+    description: Population with access to improved sanitation
+    unit: million
+    shape: Population|Sanitation Access
```

Comment:
Reference to "shape" (or other projects) can be omitted if the variable name did not change. This is only intended for automated mapping across projects.
```suggestion
```